bitnami/external-dns/values.yaml

```diff
@@ -144,8 +144,6 @@ azure:
   ## When a secret to load azure.json is not specified,
   ## the host's /etc/kubernetes/azure.json will be used
   ##
-  ## Deprecated: please use tenantId, subscriptionId, aadClientId and aadClientSecret values instead.
-  ##
```

Comment:
Why remove the deprecation notice?

Reply:
What exactly is deprecated here? The secret? That's actually now the only way to go.

Reply:
I think it means that it is preferred to set these values so the credentials are generated from them:
https://github.com/bitnami/charts/blob/master/bitnami/external-dns/templates/_helpers.tpl#L216

Reply:
Ah ok. i've reinstated them.